composer.json

```diff
@@ -450,8 +450,8 @@
             "drupal/inline_entity_form": {
                 "Add a setting to enable/disable inline editing of existing entities (https://www.drupal.org/project/inline_entity_form/issues/2913571#comment-12811088)": "https://www.drupal.org/files/issues/2018-10-11/ief_edit_existing_2913571_7.patch"
             },
-            "drupal/iframe_resizer": {
-                "Missing license URL (https://www.drupal.org/project/iframe_resizer/issues/3385378)": "https://www.drupal.org/files/issues/2023-09-05/iframe_resizer-missing_license_url-3385378-3.patch"
+            "drupal/paragraphs": {
+                "Nested paragraphs title gets cut off (https://www.drupal.org/project/paragraphs/issues/3246140)": "https://www.drupal.org/files/issues/2024-03-11/paragraphs-3246140-19-fix_paragraph_title_cutoff.patch"
             },
```

Comment:
Please confirm that we intend to remove a patch and add a patch. Its hard for me to know.

Reply:
Yes, the iframe_resizer patch was duplicate in composer.json so we removed one of them. The paragraphs was updated and patched to solve the issue https://www.drupal.org/project/paragraphs/issues/3246140